packages/netlify-cms-backend-github/src/API.ts

```diff
@@ -859,12 +861,37 @@ export default class API {
     const resp: { statuses: GitHubCommitStatus[] } = await this.request(
       `${this.originRepoURL}/commits/${sha}/status`,
     );
-    return resp.statuses.map(s => ({
-      context: s.context,
-      target_url: s.target_url,
-      state:
-        s.state === GitHubCommitStatusState.Success ? PreviewState.Success : PreviewState.Other,
-    }));
+    const deploys: GitHubDeployments = await this.request(
+      `${this.originRepoURL}/deployments?task=deploy&ref=${sha}`,
```

Comment:
I assume this returns an empty array when there are no deploys?

Reply:
Yes!